src/locales/en.json

```
@@ -619,6 +619,17 @@
   "modal": {
     "close": "Close"
   },
+  "error-images": {
+    "depressed-musician": "A depressed pianist rests their head in their hands.",
+    "waiting-for-a-bite": "Three boys sit on a broken log while two of them fish."
+  },
+  "no-results": {
+    "heading": "We couldn't find anything for \"{query}\".",
+    "alternatives": "Try a different search or see the additional sources."
```

```suggestion
    "alternatives": "Try a different search or see the additional sources."
```

Something about "see the additional sources" doesn't seem quite right to me, but I'm struggling to come up with alternative text.

---

"Try a different query or use one of the additional sources to expand your search." Maybe something along those lines?

---

I like the first suggestion too, it's clear but not too verbose. Fixed in 1d33193b.